controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -607,8 +607,8 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
     std::shared_ptr<wfa_map::tlvApRadioBasicCapabilities> radio_basic_caps = nullptr;
     std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1                         = nullptr;
     bool intel_agent                                                       = false;
-    while (1) {
-        int type = cmdu_rx.getNextTlvType();
+    int type = cmdu_rx.getNextTlvType();
+    do {
```

Comment:
Why did you choose to use do-while instead of regular while?
In do-while, you assume that the CMDU must have TLVs. But hypothetically, what will happen if the first TLV is EndOfMessage (CMDU has no TLVs)

Reply:
Can't happen - Autoconfig CMDU contains at least 2 TLVs. But since both you and Arnout prefer the `while` pattern, I changed it to your liking.